lambda/logserver/logserver.go

```diff
@@ -118,7 +121,13 @@ func (ls *LogServer) handler(res http.ResponseWriter, req *http.Request) {
 		switch event.Type {
 		case "platform.start":
 			ls.lastRequestIdLock.Lock()
-			ls.lastRequestId = event.Record.(map[string]interface{})["requestId"].(string)
+			switch event.Record.(type) {
+			case map[string]interface{}:
+				ls.lastRequestId = event.Record.(map[string]interface{})["requestId"].(string)
```

Comment:
Is the casting necessary here? The case determines it's a map.

Reply:
It can be either a map or a string.

Reply:
Both of these casts seemed necessary to pass editor linting. There may be another way to avoid them, but IntelliJ seemed insistent.